README.md

```diff
@@ -151,17 +172,106 @@ tailscale up --advertise-routes="10.0.0.0/24,192.168.0.0/24"
 ```
 
 ### Can I route traffic from machines on my local network to Tailscale endpoints automatically?
-In theory, yes - however it does require manual changes to your routing rules and these will need
-to be updated if you take advantage of WAN fail-over. This has been discussed in more detail
-[here](https://github.com/SierraSoftworks/tailscale-udm/discussions/51).
 
-*Note that we do not currently include this in `tailscale-udm` due to the risk of breaking conflicts in future.*
+Yes! As of January 30, 2025, [two changes][1][2] to Tailscale have made this
+possible. Much credit goes to @tomvoss, who contributed significant effort to
+the initial implementation, detailed [here][tailnet-routing-discussion].
+Before going further please read tailscale's [subnet router
+documentation][tailscale-subnet-router-docs] and familiarize yourself with the
+concepts of subnet routers, independent of Unifi OS.
+
+#### Prerequisites
+
+Before proceeding, please review Tailscale’s [subnet router
+documentation][tailscale-subnet-router-docs] to understand the core concepts of
+subnet routing, independent of UniFi OS.
+
+Note: You do not need to manually enable `net.ipv4.ip_forward` on your UniFi OS
+device—it is enabled by default. If you want to confirm its status, run:
+
+```sh
+sysctl net.ipv4.ip_forward
+```
+
+#### Installation and Configuration
+
+If Tailscale is already installed, update its configuration:
```

Comment:
A few little gotchas which might catch folks out:

- This should really be executed over a direct connection to the UDM (not via Tailscale, since the connection will be reset and if there is an issue, you won't be able to get back in easily to resolve it)
- Running the install script again will reset the `FLAGS`, so we can probably just use the same approach (`TAILSCALED_FLAGS="..." ./install.sh`) for both scenarios.

Reply:
I tried this, but my `/etc/default/tailscaled` is still the same (user space networking)

```
root@Ruteren:~# export TAILSCALED_FLAGS="--socket=/var/run/tailscale/tailscaled.sock"
root@Ruteren:~# curl -sSLq https://raw.github.com/SierraSoftworks/tailscale-udm/main/install.sh | sh
Tailscale is already installed and running, if you wish to update it, run '/data/tailscale/manage.sh update'
If you wish to force a reinstall, run '/data/tailscale/manage.sh install!'
Tailscaled started successfully
Run tailscale up to configure the interface.
```

Reply:
@notheotherben I added a warning and added a persistent note regarding `/data/tailscale/tailscale-env`

Reply:
Thanks! I was looking for `/data/tailscale/tailscale.env` initially. Now I found `/data/tailscale/tailscale-env` and have updated it per your instructions. Much appreciated!